components/root_layout/root_layout_body.vue

```diff
@@ -15,7 +15,7 @@
     <main
       v-if="$slots.content"
       :class="['root-layout__content', contentClass]"
-      :style="{ 'min-inline-size': contentWrapWidthPercent }"
+      :style="{ 'min-inline-size': contentWrapWidthPercent, 'height': fixed ? mainHeight : null }"
```

Comment:
Would like a refactor on the components to avoid empty style/class attributes (This doesn't affect the proper working but it is a more presentable component).
Also to avoid large style attributes or logic in the template.

Template should be cleaner as possible.

Reply:
Yep good call, really no need for this logic to be in the template, it can just go in the mainHeight computed property.